oidc-consumer/src/index.ts

```diff
@@ -92,21 +98,26 @@ class OidcConsumer {
    * @throws 400 - Missing Callback URL
    */
   async authRedirect(request: Request, response: Response, queryParams?: Object) {
-    const { redirectUri: requestRedirectURI } = request.query;
+    const { redirectUri: destination } = request.query;
 
-    if (!requestRedirectURI && !this.callback_url && !this.callback_route) return response.status(400).json({ message: "Missing Callback URL" });
+    if (!destination && !this.callback_url && !this.callback_route) return response.status(400).json({ message: "Missing Callback URL" });
```

Comment:
I think this if check needs an update. Perhaps only check for destination, since other variable are now defaults?